test/unit/org/apache/cassandra/cql3/ast/Mutation.java

```diff
@@ -510,12 +510,25 @@ protected BaseBuilder(Kind kind, TableMetadata table)
             this.primaryColumns.addAll(clusteringColumns);
             this.regularAndStatic = new LinkedHashSet<>();
             this.regularAndStatic.addAll(toSet(table.regularAndStaticColumns()));
-            this.allColumns = toSet(ASTGenerators.allColumnsInFixedOrder(table));
+            this.allColumns = toSet(safeColumns(table));
             neededPks.addAll(partitionColumns);
         }
 
+        public static List<ColumnMetadata> safeColumns(TableMetadata metadata)
```

Comment:
Change to `allColumnsInSelectOrder`?

Reply:
you had the same comment with `ASTGenerators.safeColumns`... let me unify this stuff

Reply:
pushed this to `TableMetadata` and removed this function and the one in `ASTGenerators`